src/test/java/uk/gov/companieshouse/company/links/processor/InsolvencyStreamProcessorTest.java

```diff
@@ -63,21 +63,6 @@ void successfullyProcessResourceChangedDataInsolvencyLinksGetsUpdated() throws I
 
 
         insolvencyProcessor.processDelta(mockResourceChangedMessage);
-
-        verify(companyProfileService).getCompanyProfile(CONTEXT_ID, MOCK_COMPANY_NUMBER);
-        verify(logger, times(2)).trace(anyString());
-        verify(logger, atLeastOnce()).trace(
-                contains("Resource changed message with contextId context_id of kind company-insolvency"));
-        verify(logger, atLeastOnce()).trace(String.format(
```

Comment:
Not sure why we removing all these assertions?

Reply:
They are pointless only asserting the log messages at trace level which are not a requirement.

Reply:
Agree but by mistake you removed valid assertion as well.

verify(companyProfileService).getCompanyProfile(CONTEXT_ID, MOCK_COMPANY_NUMBER);